exp/operator/controllers/singleton_installer.go

```diff
@@ -0,0 +1,80 @@
+package controllers
+
+import (
+	"sync/atomic"
+)
+
+type InstallStatus int32
+
+const (
+	InstallStatusUnknown    InstallStatus = iota
+	InstallStatusInstalling InstallStatus = iota
+	InstallStatusReady      InstallStatus = iota
```

Comment:
Don't you typically just have the list with no equals when using iota? I think both work, not sure if there's a preference
```suggestion
	InstallStatusUnknown    InstallStatus = iota
	InstallStatusInstalling InstallStatus
	InstallStatusReady      InstallStatus
```